src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -797,26 +807,22 @@ export default class CanvasController {
         cc.save();
 
         if (!aircraft.inside_ctr) {
-            cc.fillStyle = this.theme.RADAR_TARGET_OUTSIDE_RANGE;
+            cc.fillStyle = this.theme.RADAR_TARGET.HISTORY_DOT_OUTSIDE_RANGE;
         } else {
-            cc.fillStyle = this.theme.RADAR_TARGET_IN_RANGE;
+            cc.fillStyle = this.theme.RADAR_TARGET.HISTORY_DOT_INSIDE_RANGE;
         }
-
         const length = aircraft.relativePositionHistory.length;
         for (let i = 0; i < length; i++) {
```

Comment:
added `const positionHistory` to add some clarity. I agree we could use x and y properties rather than a 2 element array here, but the convention absolutely everywhere else is the 2 element array, and until they are all dealt with in #236, it would only serve to make the canvas controller more difficult to work with by fixing that in only one place.